File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.cpp
class Solution {
public:
    int longestConsecutive(vector<int>& nums) {
        if (nums.empty()) {
            return 0;
        }
        set<int> ordered_s;
        for (int num : nums) {
            ordered_s.insert(num);
        }
        int length= 1;
        int maxLength= 1;
        for (auto it= ordered_s.begin(); it != ordered_s.end(); ++it) {
            auto next_it= next(it);
            if ((next_it != ordered_s.end()) && (*it == *next_it - 1)) {
                length++;
            } else {
                if (length > maxLength) {
                    maxLength= length;
                }
                length= 1;
            }
        }
        return maxLength;
    }
};

File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        for i in range(len(nums)):
            if nums[i] == val:
                for j in range(i, len(nums)):
                    if nums[i] != nums[j]:
                        temp = nums[i]
                        nums[i] = nums[j]
                        nums[j] = temp
                        break
        res = 0
        for num in nums:
            if num != val:
                res += 1
            else:
                break
        return res